parser-combinators.cabal

```diff
@@ -1,5 +1,5 @@
 name:                 parser-combinators
-version:              0.1.0
+version:              0.1.1
```

Comment:
Please don't bump, I'll bump myself. If you really want to bump it should be 0.2.0, not 0.1.1.